app/src/main/java/com/omkarmoghe/pokemap/views/LoginActivity.java

```diff
@@ -254,5 +271,56 @@ public void onAnimationEnd(Animator animation) {
             mLoginFormView.setVisibility(show ? View.GONE : View.VISIBLE);
         }
     }
+
+    private void triggerAutoLogin() {
+
+        showProgress(true);
+
+        if (mPref.isUsernameSet() || mPref.isPasswordSet()) {
+            mNianticManager.login(mPref.getUsername(), mPref.getPassword());
```

Comment:
Hmm, don't understand... because there is an event being fired in .login() and this event transports all of the data when login fails. This event is listened on in 

    @Subscribe
    public void onEvent(final LoginEventResult result) { 

Please take a second look at it :)